Mention extractNativeLibs for Flutter/Android (#3450)
getsentry/sentry-docs: src/platforms/flutter/upload-debug.mdx

##  Uploading for Android NDK

> <Note>
> 
> Native symbolication on Android requires a specific configuration described on the [troubleshooting](/platforms/flutter/troubleshooting/#native-symbolication-on-android) page.
> 
> </Note>
> 

Reviewer: This location is better to surface the required set up for Android symbolication.

```suggestion
Native symbolication on Android requires a specific configuration. For good symbolication, set `android:extractNativeLibs="true"` in your `AndroidManifest.xml` file or `android.bundle.enableUncompressedNativeLibs=false` in your `gradle.properties` file.
```

##  Uploading for Android NDK

> Native symbolication on Android requires a specific configuration. For good symbolication, set `android:extractNativeLibs="true"` in your `AndroidManifest.xml` file or `android.bundle.enableUncompressedNativeLibs=false` in your `gradle.properties` file.
> If you use the [Android NDK](/platforms/android/using-ndk/), you must [upload files manually using our CLI](/product/cli/dif/#uploading-files).

Author: ```suggestion
Native symbolication on Android requires a specific configuration. For good symbolication, set `android:extractNativeLibs="true"` in your `AndroidManifest.xml` file or `android.bundle.enableUncompressedNativeLibs=false` in your `gradle.properties` file.

If you use the [Android NDK](/platforms/android/using-ndk/), you must [upload files manually using our CLI](/product/cli/dif/#uploading-files).
```

##  Uploading for Android NDK

> Native symbolication on Android requires a specific configuration. For good symbolication, set `android:extractNativeLibs="true"` in your `AndroidManifest.xml` file or `android.bundle.enableUncompressedNativeLibs=false` in your `gradle.properties` file.

Author: ```suggestion
Native symbolication on Android requires a specific configuration. For good symbolication, set `android:extractNativeLibs="true"` in your `AndroidManifest.xml` file or `android.bundle.enableUncompressedNativeLibs=false` for [Android App Bundle](https://developer.android.com/guide/app-bundle) in your `gradle.properties` file.
```
